landsat/landsat.ipynb

```diff
@@ -7,7 +7,7 @@
     "# Landsat\n",
     "Written by Brendan Collins<br>\n",
     "Created: June 20, 2016<br>\n",
-    "Last updated: May 25, 2021"
+    "Last updated: May 29, 2023"
```

Comment:
As you can see on the [dev website](https://pyviz-dev.github.io/examples/landsat/landsat.html) a header is now automatically generated, so this header is no longer required and can be removed from the notebook.

Reply:
I see Brendan's github ID ([brendancol](https://github.com/brendancol)) - I'll add this to anaconda-project.yml

Reply:
I totally don't remember who started it originally, which was presumably Brendan, but I definitely did contribute substantially to it, e.g. making the hyperspectral plots and the lookup table involved, so I should be listed as a second author.

Reply:
I already saw you as the first author in the anaconda-project.yml and just added his name to it as well. I assumed that was the correct thing to do here :)

Reply:
Sure. Github blame might resolve that, but who knows.